app/models/queue_item.rb

```diff
@@ -0,0 +1,17 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  delegate :category, to: :video
+  delegate :title, to: :video, prefix: :video
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
```

Comment:
For searching a single record - you should be using `.find_by` instead of `where(arg).first`. Kevin shows the way that was popular in Rails 3, however this changed in Rails 4. `.find_by` is more efficient for single records.